src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -135,6 +135,13 @@ internal override void Initialize(ApplicationEngine engine)
             engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_RequestId), new StorageItem(BitConverter.GetBytes(0ul)));
         }
 
+        [ContractMethod(0_00100000, CallFlags.None)]
+        private void OnPayment(ApplicationEngine engine, UInt160 from, BigInteger amount)
+        {
+            if (engine.CallingScriptHash != Hash)
```

Comment:
So, it's like "Oracle" tokens? Maybe we need to fix GAS calls to go through `CallFromNativeContract`? Although it's `Mint` being called here...

Reply:
`GAS.Mint` cann't be called by contract, so we cann't call it through `CallFromNativeContract`.